### PR TITLE
RAIN-1287: README.md and LOCALSETUP.md for egov-pdf

### PR DESCRIPTION
**Pull request by gopesh67**

### CODE REVIEW

File: egov-pdf/LOCALSETUP.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Postgres DB
+- [ ] Redis
+- [ ] Elasticsearch
+- [ ] Kafka
+  - [ ] Consumer
+  - [ ] Producer
+
+## Running Locally
+
+- Provide host url in `HOST` property in `config.js`
```
Comment (tarunlalwani) [inline egov-pdf/LOCALSETUP.md:18]: In readme, we are saying services dependencies as below

- egov-mdms-service
- tl-service
- Property-service
- Pdf-service
- User-service
- Collection service
- Billing-service
- Workflow-service

But here there is no talk about any configuration? Just confirm this once

File: egov-pdf/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- PT mutationcertificate
+- PT bill
+- PT receipt
+- TL receipt
+- TL certifcate
+- TL renewal certificate
+- Consolidated receipt
+
+#### Configurations
+NA
```
Comment (tarunlalwani) [inline egov-pdf/README.md:39]: Talk about how to add a new service pdf to this

**Review by tarunlalwani — CHANGES_REQUESTED** [review]

File: egov-pdf/README.md (hunk 2 above)
Comment (rohit-eGov) [reply egov-pdf/README.md]: Added the steps for adding support for new pdf

File: egov-pdf/LOCALSETUP.md (hunk 1 above)
Comment (rohit-eGov) [reply egov-pdf/LOCALSETUP.md]: Added the list of properties which has to be updated before running

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**